app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,35 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: Rails.configuration.folio_sync['default_sender_email_address']
+  layout 'mailer'
+
+  def folio_sync_error_email
+    body_content = "One or more errors were encountered during FOLIO sync:\n\n"
+
+    # Downloading errors contain resource_uri and error message
+    if params[:downloading_errors].present?
+      body_content += "======== Downloading Errors ======== \n"
+      params[:downloading_errors].each do |error|
+        body_content += "Resource URI: #{error[:resource_uri]}\n"
```

Comment:
We'll need a way to disambiguate Barnard and RBML URIs (which will both start with /repositories/2). This could be done by indicating the repository/instance, or by adding more resource-specific information (like collection ID or title).

Reply:
Agreed! Verifying that this will be implemented as part of ASI-299 (handling different instances)